package.json

```diff
@@ -82,16 +82,18 @@
   ],
   "jspm": {
     "main": "js/bootstrap",
+    "ignore": ["dist/js/npm"],
+    "directories": {
+      "lib": "dist"
```

Comment:
Not sure what your point is- I'll defer to you and @guybedford on this of course- let me know if you guys want me to amend the PR.

Let me clarify what I meant here:

>I can't think of a solid reason people would want the raw ES6 files- did you have a use case in mind?

jspm supports a variety of module formats, which makes it very easy to consume libraries, without concern for what format they're distributed in.  I think the only time I would be concerned is if the library was distributed in ES6/ES7 format and I had to transpile it as part of my build process (or worse, using the loader).

In other words I think it's an edge case for someone to say *"oh man I'm a super cutting edge guy and I only use ES6/ES7 modules (except for that pesky jQuery dependency which I'm ok with)".*  

Does this make sense?  If there is a compelling reason to distribute the ES6 bits, you might consider taking the approach the Aurelia repos take where the dist folder has subfolders for the amd, commonjs, es6 and system formats.

Thoughts?